src/Core/src/Platform/Android/EditorExtensions.cs

```diff
@@ -0,0 +1,35 @@
+using Android.Content.Res;
+using AndroidX.AppCompat.Widget;
+
+namespace Microsoft.Maui
+{
+	public static class EditorExtensions
+	{
+		public static void UpdatePlaceholder(this AppCompatEditText editText, IEditor editor)
```

Comment:
I notice this is a duplicate function, and also will work for entry. Maybe merge this file with `EditTextExtensions`